app/javascript/dashboard/components/layout/sidebarComponents/SecondaryNavItem.vue

```diff
@@ -137,7 +148,25 @@ export default {
     isAllArticles() {
       return (
         this.$store.state.route.name === 'list_all_locale_articles' &&
-        this.menuItem.toStateName === 'all_locale_articles'
+        this.menuItem.toStateName === 'list_all_selectedPortalLocale_articles'
+      );
+    },
+    isMyArticles() {
+      return (
+        this.$store.state.route.name === 'list_mine_articles' &&
+        this.menuItem.toStateName === 'mine_articles'
```

Comment:
Is it mine_articles or list_mine_articles? 

Also, this code seems to be repeated below. Can we create a method?